### PR TITLE
Create DNF Sort File

### PR DESCRIPTION
**Pull request by Pranshu321**
## Related Issue
- #119

## Proposed Changes
- DNF Sort code and full explanation provided. 

## Additional Info
- In this, I deeply explain about the DNF sort how it works.
- What's its time and Space Complexity?
- And its Approach and Algorithm.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

In this pull request, all the things which are required by the beginner for understanding the **DNF Sort** all things are defined in great detail, with simple language, and all points are covered.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
| Title goes here  | Title goes here     |
| Image goes here  | Image goes here     |

### CODE REVIEW

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]
add comments to your code

File: S-SortingAlgorithms/README.md [hunk 1]
```diff
@@ -85,6 +86,34 @@ end BubbleSort
...
+
+## Approach
+
+**The Array is divided into four Sections**
+
+1. `a[1 to low-1]` for zeroes and as repect to flag (red in Dutch Flag).
+2. `a[low to mid-1]` for ones and as repect to flag (white).
+3. `a[mid to high]` no change.
+4. `a[high+1 to n(size)]` for twos and as respect to flag (blue).
+
```
Comment (Lakhankumawat) [inline S-SortingAlgorithms/README.md:100]: Here add a working image of your sorting algorithm
You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by Lakhankumawat** [comment]
### 1. Starr the REPO

**Comment by miraj0507** [comment]
This looks good to me

**Comment by Pranshu321** [comment]
> This looks good to me

@miraj0507 sir, Please check now, and really sorry for late response.

**Comment by Pranshu321** [comment]
DNF sort algorithm is already present.

**Pull request closed without merging**